### PR TITLE
Add content to issue JSON

### PR DESCRIPTION
**Pull request by grstk**
TODO: add actual contents for ClassLength, ModuleLength and AbcSize checks.

### CODE REVIEW

**Comment by mrb** [comment]
Needs review please /cc @codeclimate/review

File: lib/cc/engine/rubocop.rb [hunk 1]
```diff
@@ -83,17 +85,25 @@ def violation_positions(location)
...
+                            type: "Issue",
+                            check_name: "Rubocop/#{violation.cop_name}",
+                            description: violation.message,
+                            categories: [category(violation.cop_name)],
+                            remediation_points: 50_000,
+                            location: {
+                              path: local_path,
+                              positions: violation_positions(violation.location),
+                            },
+                         }
```
Comment (pbrisbin) [inline lib/cc/engine/rubocop.rb]: Can you format this as:

```rb
violation = {
  type: ...,
  check_name: ...,
}
```

That's our style and will make the diff clearer too.

File: spec/cc/engine/rubocop_spec.rb [hunk 2]
```diff
@@ -201,10 +201,27 @@ def method(a,b,c,d,e,f,g)
         assert_equal location, result["location"]
       end
 
+      it 'includes issue content when available' do
```
Comment (pbrisbin) [inline spec/cc/engine/rubocop_spec.rb]: :lipstick: please use double quotes throughout.

**Comment by pbrisbin** [comment]
Code LGTM (modulo some minor nits). I don't think we can merge until the TODO is addressed though.

**Comment by GordonDiggs** [comment]
Is the plan that we're going to copy and paste in content for every cop? The classes define [their own descriptions](http://www.rubydoc.info/gems/rubocop/0.27.0/RuboCop/Cop/Metrics/AbcSize), though it is as a comment in the code. Is there a way we could interface with that? Perhaps by parsing the YARD documentation and accessing it that way?

**Comment by GordonDiggs** [comment]
Copying and pasting doesn't seem like a very sustainable solution, is all

**Comment by grstk** [comment]
@GordonDiggs I don't think the content for check is supposed to be the same as cop definition from comments (I was told that the contents for the three checks above were 'previously written'). @brynary can you confirm?

**Comment by pbrisbin** [comment]
He probably mean't to bring in the existing ReadUp content for the similar complexity checks we have in non-platform analysis.

@mrb can you confirm? If so, would you be able to find the correct content in admin/content_blobs and copy them in here?

**Comment by brynary** [comment]
What @pbrisbin said is correct.

**Comment by mrb** [comment]
Yep I got it @pbrisbin

**Pull request closed without merging**